### DISCUSSION TITLE
Connection dying after 10 minutes

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1725] [Developer Support]
"Hi all! Is there any timeout for the WC connection? I am building a wallet, I connect to opensea, and about 10 minutes after the connection, it just stops receiving transactions. Any clues? Thanks!"

Dorara, "I have same issue when connecting after like 5-10 min
walletConnectV2swift socket connection is lost.
Im sure I using Networking.configure(projectId: projectID, socketFactory: SocketFactory()) with .automatic"

**Comment by chadyj** [comment]
There are 2 different issues here.

OpenSea is using v1, and that is deprecated.

The second comment is about v2. Please submit a bug report https://github.com/WalletConnect/WalletConnectSwiftV2/issues/new/choose
